commitlint.config.js

```diff
@@ -3,5 +3,13 @@ module.exports = {
   ignores: [
     (commit) => commit.includes('[CI-SKIP]'),
     (commit) => commit.includes('Pull request'),
+    /**
+     * Ignore Dependabot commits because they are violating sentence case rule
+     * and cannot be configure or respect commitlint configuration.
```

Comment:
_cannot be configured or forced to respect …_, or similar

Reply:
Fixed.